tools/codemods/src/index.ts

```diff
@@ -14,13 +15,46 @@ export interface MigrateOptions {
   force?: boolean;
   ignore?: Array<string>;
   packages?: Array<LGPackage>;
+  list?: boolean;
 }
 
 export const migrator = async (
   codemod: string,
   files?: string | Array<string>,
   options: MigrateOptions = {},
 ) => {
+  if (options.list || codemod === 'list') {
+    // If list option is true, (or the provided codemod name is "list")
+    // show available codemods and exit
+    const codemods = await getAvailableCodemods();
+
+    if (codemods.length === 0) {
+      console.log(chalk.yellow('No codemods found'));
+    } else {
+      console.log(chalk.greenBright('Available codemods:'));
+      codemods.forEach(cm => {
+        console.log(`- ${chalk.cyan(cm)}`);
+      });
+      console.log(
+        `\nFor more details visit: `,
+        chalk.gray(
+          '\nhttps://github.com/mongodb/leafygreen-ui/blob/main/tools/codemods/README.md#codemods-1',
+        ),
+      );
+    }
+
+    return;
+  }
+
+  // Check if a valid codemod name is provided
+  if (!codemod || codemod.trim() === '') {
+    console.error(chalk.red('Error: No codemod specified'));
+    console.log(
+      chalk.yellow('Use the --list option to see available codemods.'),
```

Comment:
[There's a console log a few lines below](https://github.com/mongodb/leafygreen-ui/blob/98d281c2ac418d2bc7d25279cc567d230c1f8072/tools/codemods/src/index.ts#L68-L72) that probably could be updated to use the `--list option`